### ISSUE TITLE
duplicate empty opencv2.pc pkgconfig file installed in groovy

### ISSUE THREAD

**Issue by meyerj** [ros-perception/vision_opencv#11]
I am not sure if this is related to the vision_opencv stack, but as I did not find a separate bug tracker for the ROS packaged version of opencv2, I report it here:

The current Ubuntu package ros-groovy-opencv2 (version 2.4.3-6precise-20130125-0338-+0000) installs to pkgconfig files in /opt/ros/groovy/lib/pkgconfig, namely opencv.pc and opencv2.pc. The second file seems to be malformed, as it does not contain any includes or library definitions.

opencv.pc:

```
# Package Information for pkg-config

prefix=/opt/ros/groovy
exec_prefix=${prefix}
libdir=
includedir_old=${prefix}/include/opencv
includedir_new=${prefix}/include

Name: OpenCV
Description: Open Source Computer Vision Library
Version: 2.4.3.2
Libs: -L${exec_prefix}/lib -lopencv_calib3d -lopencv_contrib -lopencv_core -lopencv_features2d -lopencv_flann -lopencv_gpu -lopencv_highgui -lopencv_imgproc -lopencv_legacy -lopencv_ml -lopencv_nonfree -lopencv_objdetect -lopencv_photo -lopencv_stitching -lopencv_ts -lopencv_video -lopencv_videostab
Cflags: -I${includedir_old} -I${includedir_new}
```

opencv2.pc:

```
Name: opencv2
Description: Description of opencv2
Version: 2.4.3
Cflags: -I
Libs: -L/opt/ros/groovy/lib
Requires:
```

**Comment by vrabaud** [comment]
Weird, thx for reporting. That is the best place for reporting thx (I just updated the wiki page to reflect that).
opencv2.pc is created by catkin and opencv.pc is the official one. There is indeed a problem.
Can you please state your use case so that we fix it better ? Where do you use the pkgconfig file ? thx

**Comment by meyerj** [comment]
I did not use the file directly nor observed any problems with the second file. I just stumbled across the file while searching for possbile causes for the issue reportet in #12. As I assumed the two issues are not related to each other, I reported them separately.

At a closer look, the linker error from #12 could be caused by an errorneous package manifest invoking `pkg-config opencv2` instead of `pkg-config opencv`, but I did not find one in my installed fuerte stacks.

**Comment by meyerj** [comment]
If I understand the catkin documentation right, this issue could be resolved by adding `catkin_package(... DEPENDS opencv)` to the CMakeLists.txt script for the opencv2 package. catkin should add the dependeny from package opencv in the `Requires:` section of the catkin-generated opencv2.pc file, so using both would be fine as pkg-config resolves the dependency internally.

**Comment by vrabaud** [comment]
ok, that got fixed with REP 136 that I implemented in OpenCV 2.4.3-7. Thx for reporting !
